error[E0080]: evaluation of `PrintName::<i32>::VOID` failed
  --> $DIR/erroneous_const.rs:10:21
   |
10 |     const VOID: ! = panic!(); //~ERROR evaluation of `PrintName::<i32>::VOID` failed
   |                     ^^^^^^^^ the evaluated program panicked at 'explicit panic', $DIR/erroneous_const.rs:10:21
   |
   = note: this error originates in the macro `$crate::panic::panic_2015` (in Nightly builds, run with -Z macro-backtrace for more info)

error: error occurred: encountered constants with type errors, stopping evaluation
  --> $DIR/erroneous_const.rs:15:17
   |
15 |         let _ = PrintName::<T>::VOID; //~ERROR error occurred: encountered constant
   |                 ^^^^^^^^^^^^^^^^^^^^ encountered constants with type errors, stopping evaluation
   |
   = note: inside `no_codegen::<i32>` at $DIR/erroneous_const.rs:15:17
note: inside `main` at $DIR/erroneous_const.rs:19:5
  --> $DIR/erroneous_const.rs:19:5
   |
19 |     no_codegen::<i32>();
   |     ^^^^^^^^^^^^^^^^^^^

note: some details are omitted, run with `MIRIFLAGS=-Zmiri-backtrace=full` for a verbose backtrace

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0080`.
